bundle/config/target.go

```diff
@@ -35,6 +35,8 @@ type Target struct {
 	Git Git `json:"git,omitempty"`
 
 	RunAs *jobs.JobRunAs `json:"run_as,omitempty"`
+
+	Sync Sync `json:"sync,omitempty"`
```

Comment:
Please make this a pointer; then we can really omit it from the validate output if not set.

As a value type, it will always be included as an empty object because "emptiness" cannot be determined.